frinx-uniconfig/user-guide/network-management-protocols/uniconfig_gnmi/readme.md

```diff
@@ -0,0 +1,208 @@
+to CLI or NETCONF southbound plugins. However, the difference is that it uses protocol buffer (proto3) 
+for service structure modeling within it's RPCs instead of YANG.
+
+## gNMI southbound plugin
+
+The gNMI southbound plugin is capable of connecting to remote gNMI devices and exposing their 
+datastores (CONFIG, STATE) as MD-SAL mount points. These mount points allow applications and remote users 
+(over RESTCONF) to interact with the mounted devices. By mounting of gNMI device a new session is gNMI session is created.
+The gNMI session is responsible for establishing a connection via gRPC ManagedChannel. After the connection is established, 
+gNMI southbound plugin read the device capabilities via Capabilities RPC. From the capabilities, a new schema context is built 
+(once the schema context is built, it is cached for next runs) and all necessary services are created (DOMDataBroker, 
+DOMRpcService, DOMNotification/Subscription service). After all of this is done, gNMI southbound plugin will mark 
+the connection status of the particular device as READY, so UniConfig layer can handle it.
+
+## Important features
+
+When gNMI device is installed, it is possible to do:
+
+- read (gNMI GET), create/update/delete (gNMI SET) the configuration of device:
+  - via UniConfig topology
+  - via gNMI souhtbound topology (using `yang-ext:mount`)
+  - via yang-patch operations over UniConfig/gNMI souhtbound topology (using `yang-ext:mount`)
+- manipulate with a device via gNOI RPCs
+- use gNMI SET update operation for specific paths
+- use gNMI SET replace operation for specific paths
+- order config changes (replace/delete operation) according to dependency paths
+- remove module-name from list keys for specific paths
+- read from ALL datastore for specific paths
+- subscribe to telemetry streaming for specific paths (gNMI Subscribe)
+- store failed installations
+- enable logging-broker for gNMI messages (SET/GET)
+- use DryRunCommit feature
+
+### gNOI 
+
+The gNOI (gRPC Network Operations Interface) defines gRPC-based microservices 
+for executing operational commands on network devices. By default gNMI southbound plugin is capable of 
+executing Certification, File, System and OS RPCs according to protobuf files <https://github.com/openconfig/gnoi>. 
+Apart from the common ones, it is possible to invoke also SONiC type gNOI RPCs by specifying 
+`"gnmi-topology:device-type" : "sonic"` in install request inside of `connection-parameters` container. 
+It's necessary to have `gnoi-sonic@2023-03-17.yang` schema model in cache directory for this specific node, 
+to be able to invoke it via RESTCONF. 
+
+Example of gNOI RPC invocation
+------------------------------
+
+```bash Invocation of SONiC specific copy-config RPC
+curl --location --request POST 'http://localhost:8181/rests/operations/network-topology:network-topology/topology=gnmi-topology/node=sonic/yang-ext:mount/gnoi-sonic:copy-config' \
+--header 'Authorization: Basic YWRtaW46YWRtaW4=' \
```

Comment:
Authorization header is not used anymore

Reply:
ack